src/main/features/response/views/full-rejection/how-much-have-you-paid.njk

```diff
@@ -0,0 +1,31 @@
+{% extends "layout.njk" %}
```

Comment:
Nasty duplicate :( Can you please create story to get rid of it after Wed?

Reply:
ROC-4282